3_javascript/javascript1/week1/script.js

```diff
@@ -0,0 +1,72 @@
+let yearOfBirth = 1998;
+let yearFuture = 2023;
+
+// calculat age 
+let age = yearFuture - yearOfBirth; 
+let stringOutput = `You will be ${age} years old in ${yearFuture} .`
+// log age 
+console.log(stringOutput);
+
+
+// A dog age calculator
+let dogYearOfBirth = 2010;
+let dogYearFuture = 2027;
+
+let shouldShowResultInDogYears = true;
+let dogAgeStringOutput;
+
+if (shouldShowResultInDogYears) {
+    let dogAgeInDogYears = (dogYearFuture - dogYearOfBirth) * 7
+    dogAgeStringOutput = `Your dog will be ${dogAgeInDogYears} dog years old in ${dogYearFuture}`;
+  } else {
+    let dogAgeInHumanYears = dogYearFuture - dogYearOfBirth;
+    dogAgeStringOutput = `Your dog will be ${dogAgeInHumanYears} human years old in ${dogYearFuture}`;
+  }
+
+console.log(dogAgeStringOutput);
+// House price 
+
+let gardenSizeInM2 = 100;
+function housePrice(volumeInMeters,gardenSizeInM2){
```

Comment:
A more conventional what would be to call this getHousePrice (or caclulateHousePrice) - it is then instantly clear what this function is for. 
A name like housePrice is more suited for a variable that, unlike a function, does not _do_ anything.

Reply:
i see, i think i will go with getHousePrice, it sounds clearer as you mentioned